app/MMB/src/components/menu/MenuItem.js

```diff
@@ -54,37 +59,48 @@ type Props = {|
 function MenuItem(props: Props) {
   const styleSheet = createStyleSheet(props);
 
-  return (
-    <Touchable onPress={props.onPress} style={styleSheet.wrapper}>
-      <React.Fragment>
-        {props.icon && (
-          <MenuItemIcon
-            iconComponent={props.icon}
-            invertColors={props.isActive}
-          />
-        )}
-
-        <View style={styleSheet.middleWrapper}>
-          <Text style={styleSheet.title}>{props.title}</Text>
-          {props.description && (
-            <Text style={styleSheet.description}>{props.description}</Text>
-          )}
+  const IconRight = props.actionIcon ? (
+    <View style={styleSheet.rightArrow}>{props.actionIcon}</View>
+  ) : (
+    <AdaptableLayout.Consumer
+      renderOnNarrow={
+        <View style={styleSheet.rightArrow}>
+          <Icon name="chevron-right" size={26} color={Color.brand} />
         </View>
+      }
+    />
+  );
 
-        {props.actionIcon ? (
-          <View style={styleSheet.rightArrow}>{props.actionIcon}</View>
-        ) : (
-          <AdaptableLayout.Consumer
-            renderOnNarrow={
-              <View style={styleSheet.rightArrow}>
-                <Icon name="chevron-right" size={26} color={Color.brand} />
-              </View>
-            }
-          />
+  const MenuItemInner = (
+    <React.Fragment>
+      {props.icon && (
+        <MenuItemIcon
+          iconComponent={props.icon}
+          invertColors={props.isActive}
+        />
+      )}
+
+      <View style={styleSheet.middleWrapper}>
+        <Text style={[styleSheet.title, props.titleStyle]}>{props.title}</Text>
+        {props.description && (
+          <Text style={[styleSheet.description, props.descriptionStyle]}>
+            {props.description}
+          </Text>
         )}
-      </React.Fragment>
-    </Touchable>
+      </View>
+      {/* Hide icon if menu item is not clickable */}
+      {props.onPress && IconRight}
+    </React.Fragment>
   );
+
+  if (props.onPress) {
+    return (
+      <Touchable onPress={props.onPress} style={styleSheet.wrapper}>
```

Comment:
There is also `disabled` property but it requires dummy `onPress` anyway.

Reply:
I think we already use similar approach in `SimpleCard`